suds/reader.py

```diff
@@ -49,7 +49,7 @@ def mangle(self, name, x):
         Mangle the name by hashing the I{name} and appending I{x}.
         @return: the mangled name.
         """
-        h = hashlib.md5(name.encode('utf8')).hexdigest()
+        h = hashlib.sha256(name.encode('utf8')).hexdigest()
```

Comment:
Can we use **sha1** algorithm instead of using **sha256**, sha256 algorithm has the digest length greater than sha1 algorithm which is not required.